kubernetes/helm-charts/buildfarm/values.yaml

```diff
@@ -131,22 +131,25 @@ shardWorker:
     type: ClusterIP
     port: 8982
 
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
-  extraVolumes: []
-    # - name: additionalSecret
-    #   secret:
-    #   secretName: my-secret
-    #   defaultMode: 0600
+  #nodeSelector: {}
+  #tolerations: []
+  #affinity: {}
+
+  #extraVolumes:
+  # - name: additionalSecret
+  #   secret:
+  #   secretName: my-secret
+  #   defaultMode: 0600
+
+  #extraVolumeMounts:
+  # - name: customConfig
+  #   mountPath: /mnt/config
+  #   readOnly: true
```

Comment:
This looks similar to your other PR #1604, is this extra/intended, and will the template resolution treat it kindly to have these fields missing?